1_fetch.R

```diff
@@ -0,0 +1,34 @@
+# Get spatial data into sf objects
+
+source("1_fetch/src/maps_to_sf.R")
+
+p1_targets <- list(
+  
+  # Albers Equal Area
+  tar_target(p1_proj_str, "+proj=aea +lat_1=29.5 +lat_2=45.5 +lat_0=37.5 +lon_0=-96 +x_0=0 +y_0=0 +ellps=GRS80 +datum=NAD83 +units=m +no_defs"),
+  
+  tar_target(
+    p1_conus_sf,
+    generate_conus_sf(p1_proj_str)
+  ),
+  
+  tar_target(
+    p1_conus_states_sf,
+    generate_conus_states_sf(p1_proj_str)
+  ),
+  
+  # Get basins
+  # TODO: add more than the one IWS basin and propogate these
+  # labels through to the SVG additions.
+  tar_target(
+    p1_huc8s, c("07120001", "07120002", "07120003")
+  ),
```

Comment:
So if we wanted to make a map with all HUCs we would need to list them all?

Reply:
I want to make a helper function for this - allow you to pass in specific HUC 8s or get all. But currently, yes you would. Just don't have time to outfit the pipeline to do all the things just yet :)

Reply:
Made an issue. See #2